04LessonFour/in-class-project/index.html

```diff
@@ -2,72 +2,89 @@
+    <link rel="stylesheet" href="css/style.css">
 
-<body>
+  </head>
 
+  <body>
+    <!--Still curious about fonts. These do not match the previous page, but replicating it was not as simple as imagined. Change later when more fonts are installed.  -->
     <header>
 
-        <h1>Intel</h1>
-        <nav>
-            <a href="www.intel.com">Intel</a>
-            <a href="www.intel.com"> Processor</a>
-        </nav>
+      <h1>Intel&reg; </h1>
+
+      <nav>
+
+        <a href="http://www.intel.com/content/www/us/en/processors/core/core-i7-processor.html" alt="Intel Core i7 Procesor">i7 Processor</a>
+        <a href="http://www.intel.com/content/www/us/en/solid-state-drives/solid-state-drives-ssd.html" alt="Solid State Drives">Solid State Drives</a>
+
+      </nav>
 
     </header>
+    <!--Main is a container for formatting and style  -->
+    <main>
+
+
+      <!--Each section has an id for organizational purposes  -->
+      <section id="processor">
+
+        <h2>Intel &reg; Core&trade; i7</h2>
+
+        <p>The 6th Generation Intel &reg; Core&trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even
+          the toughest of tasks and games.</p>
+
+      </section>
 
-    <section id="sec_processor">
+      <section id="ready">
 
-        <h2>Intel® Core™ i7</h2>
-        <p>The 6th Generation Intel® Core™ i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even the toughest of tasks and games.</p>
-        <table>
-    </section>
+        <img id="proc" src="img/processor.jpg" alt "processor" />
+        <!--Is it acceptable to use the br tag to create space? It seems easier than padding but will change if it isn't the best coding practice. -->
+        <br />
+        <br />
 
-    <section>
-        <img src="img/processor.jpg" align="right" display="inline-block" />
-        <div class="sec_ready">
+        <h2>Ready. Wake. Create.</h2>
 
-            <h3>Ready. Wake. Create.</h3>
-            <p>From zero to power in a flash, the 6th Generation Intel® Core™ processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever.</p>
+        <p>From zero to power in a flash, the 6th Generation Intel &reg; Core&trade; processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever.</p>
 
-        </div>
-    </section>
+      </section>
 
-    <section id="sec_support">
-        <h2>Intel Customer Support</h2>
-        <p>Find Support Topics by Product, Drivers & Software, or ask the community <a href="#">read more</a></p>
+      <aside>
+        <!--Not sure if this is the best use of an aside, but since it's not part of the main content it will be left there for now -->
 
```

Comment:
if it makes sense to you, use it. remember, they are strictly semantic elements.